lib/safira/accounts/staff.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Safira.Accounts.Staff do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias Safira.Accounts.User
+
+  schema "staffs" do
```

Comment:
The plural is staff

Reply:
True but i think for readability sake we should keep it as staffs

Reply:
I think it's known to be staff no need for extra s